src/scripts/main.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
-const people = require('./lib/people');
+const usersData = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard tbody');
 
-// write your code here
+function checkSex(sex) {
+  return sex === 'm' ? 'Male' : 'Female';
+}
+
+function fillTable(people) {
+  people.forEach(person => {
+    table.insertAdjacentHTML('beforeend', `
+    <tr>
+      <td>${person.name}</td>
+      <td>${checkSex(person.sex)}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${person.died - person.born}</td>
+      <td>${Math.ceil(person.died / 100)}</td>
```

Comment:
Destructure person

Reply:
agree, will be better to get needed properties from the person object using destructuring